README.md

```diff
@@ -22,9 +21,71 @@ Or install it yourself as:
 
     $ gem install microform
 
+## Generating Forms
+
+Generate a form for your model, with accompanying tests:
+
+```
+  rails g microform:form model_name
+```
+
+This will create two files: `app/forms/model_name_form.rb` and `test/forms/model_name_test.rb`.
+
 ## Usage
 
-TODO: Write usage instructions here
+ The `Microform::Submission` module provides a `submit` method that will help to better isolate your form tests. Include it in the relevant controller for your form(s):
+
+```ruby
+  class AdminController < ApplicationController
+    include Microform::Submission
+  end
+```
+
+Then, in the controller where you want to use the form, instantiate the new form in your actions with a model instance. You can save or update attributes using the `submit` method, providing the form name, instance, and the parameters:
+
+```ruby
+  def new
+    @post = Post.new
+    @post_form = PostForm.new(@post)
+  end
+
+  def edit
+    @post_form = PostForm.new(@post)
+  end
+
+  def create
+    @post = Post.new
+    @post_form = submit(PostForm, @post, post_params)
+
+    respond_to do |format|
+      if @post_form.valid?
+        format.html { redirect_to admin_post_url(@post_form), notice: 'Post was successfully created.' }
+        format.json { render :show, status: :created, location: @post_form }
+      else
+        format.html { render :new }
+        format.json { render json: @post_form.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    @post_form = submit(PostForm, @post, post_params)
```

Comment:
It seems like `Post.find params[:id]` works, but it looks like it's usually w/ parens if the style is parens-less (at least looking through Hartl's book: https://books.google.com/books?id=ePuCDQAAQBAJ&lpg=PA1&dq=HARTL%20rails&pg=PT405#v=onepage&q&f=false)